scanner/src/main/kotlin/provenance/ProvenanceDownloader.kt

```diff
@@ -39,7 +39,7 @@ import org.ossreviewtoolkit.utils.ort.createOrtTempDir
 /**
  * An interface that provides functionality to download source code.
  */
-interface ProvenanceDownloader {
+fun interface ProvenanceDownloader {
```

Comment:
Try if FakeProvenanceDownloader can use it. If yes, make separate commit.

Reply:
The other implementation (FakeProvenanceDownloader) could be turned into a function which return an anonymous class / instance. However, this does not work with the spies used with that instance.